.github/workflows/build.yml

```diff
@@ -122,12 +122,16 @@ jobs:
           toolchain: ${{ steps.toolchain.outputs.version }}
           components: "clippy,rustfmt"
 
-      - name: Cache rust packages
+      - name: Cache rust packages / build cache
         uses: actions/cache@v3
         env:
           cache-name: solana-rust-packages
         with:
-          path: ~/.cargo/registry
+          path: |
+            ~/.cargo/bin
+            ~/.cargo/registry
+            ~/.cargo/git/db
+            solana/target
```

Comment:
Due to the way `actions/cache` works, the new directories won't get picked up until the cache key changes or we delete the current cache entry with this key from the repo.

Reply:
This also appears to significantly increase the size of the cache from 390MB -> 1.7GB.  See https://github.com/wormhole-foundation/wormhole/actions/caches